src/main/resources/api/data-repository-openapi.yaml

```diff
@@ -4659,31 +4659,31 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/ErrorModel'
-  /api/repository/v1/datasets/{id}/snapshotBuilder/concepts/{domainId}/search:
+  /api/repository/v1/snapshots/{id}/snapshotBuilder/concepts:
     get:
       tags:
-        - datasets
+        - snapshots
         - repository
       description: >
-        Searches concepts of the specified domain that match the given searchText.
-      operationId: searchConcepts
+        Retrieves concepts of the specified domain and enables filtering concepts given by the searchText.
+      operationId: enumerateConcepts
       parameters:
         - in: path
-          description: The UUID of the dataset.
+          description: The UUID of the snapshot.
           name: id
           required: true
           schema:
             type: string
             format: uuid
-        - in: path
+        - in: query
```

Comment:
why change from path params to query params? Can you give an example of a query with a `filterText`: "cancer" and domainId: 19 (Condition) 

Would that be?  

`DATAREPO_URL/api/repository/v1/snapshots/{id}/snapshotBuilder/concepts/domainId=19/filterText="cancer"`